p11-kit/modules.c

```diff
@@ -638,6 +639,71 @@ take_config_and_load_module_inlock (char **name,
 	return rv;
 }
 
+/**
+ * p11_kit_getenv:
+ * @name: Environment variable to find
+ *
+ * Returns: pointer to value of environment variable if successful
+ *
+ */
+
+static char *
+p11_kit_getenv(const char *name)
```

Comment:
We can assume that `secure_getenv` is always defined, as `common/compat.c` provides this fallback if configure/meson couldn't find it.

Reply:
Thanks -- I've removed the unnecessary check